File: basicsProgramming/module1/practice1/pascal/task1-3.pas
program Task1_3;

var
    n1, n2, n3, sum: Integer;
    avg: Real;

begin
    writeln('Введите целые числа');
    {$I-}
    readln(n1, n2, n3);
    {$I+}

    if IOResult <> 0 then
    begin
        writeln('Ошибка: введите целое значения');
        Halt(1);
    end;

    sum := n1 + n2 + n3;
    avg := sum / 3;

    WriteLn('Сумма чисел: ', sum);
    WriteLn('Среднее арифметическое: ', avg:0:6);
end.

File: basicsProgramming/module1/practice1/c/task1-3.c
#include <stdio.h>

void task1_3()
{
    int n1, n2, n3;
    long long sum;
    float avg;

    printf("Введите 3 целых числа ");
    
    if(scanf("%d %d %d", &n1, &n2, &n3) != 3){
        printf("Введите 3 целых числа");
        return;
    }

    sum = (long long)n1 + n2 + n3;
    avg = sum / 3.0;

    printf("Сумма чисел = %lld\n", sum);
    printf("Среднее арифметическое = %.2f", avg);
}

int main()
{
    task1_3();
    return 0;
}
